### ISSUE TITLE
Double specification to configure a method as a probe

### ISSUE THREAD

**Issue by hernanmd** [cormas/cormas#510]
To have a probe method listed in the Define the scenario (Confirm before launching) window, the method should be classified into the "probes" protocol.
Also the method should be annotated with a pragma "probe" to have it configured in the internals of the activeProbes mechanism.

Provide a simple UI presenter to have a probe added into the protocol, and annotated with pragma if necessary.

**Comment by pierbo** [comment]
I agree, especially since an accessor on a numeric attribute can serve as both an accessor AND a probe. So we should prefer to use Pragma
